.github/workflows/pylint.yml

```diff
@@ -0,0 +1,31 @@
+name: Pylint
+
+on:
+  push:
+      branches:
+      - main
+      - Features
+  pull_request:
+      branches:
+      - main
+      - Features
+
+jobs:
+  build:
```

Comment:
Może i czepialskie, ale dla desktyprywności powinno być tutać raczej coś innego niż build. Nic w tym jobie nie budujesz, ani nie kompilujesz.  
Lepsza byłaby chyba nazwa `pylint`, `linter` albo coś podobnego, coś co na serio mówi co tu się dzieje

Reply:
Albo `code-check`, jeśli zamierzasz w tym tasku sprawdzać także formatowanie

Reply:
mogę to zmienić, aczkolwiek to jest oficjalny workflow dystrybuowany przez github actions, podobnie jest w drugim który przeglądałem, dlatego nie wiem czy to nie jest oficjalny syntax

Reply:
Zalecam ci zajrzeć do dokumentacji, abyś zrozumiał to, co robisz https://docs.github.com/en/actions/using-workflows/about-workflows